Packs/AzureStorage/README.md

```diff
@@ -1,18 +1,18 @@
-The Azure Storage content pack enables you to manage storage accounts and blob services and blob containers.
-
-## What does this pack do?
-
-The integration in this pack enables you to:
-- Get storage account details 
-- Create or update a storage account
-- Get the blob service properties of a specific account storage. 
-- Set the blob service properties of a specific account storage
-- Create a blob container
-- Update a blob container 
-- Get all or specific blob container details.
-- Delete a blob container
-
-## Authorization
-To connect to the Azure Storage Accounts and the Blob Service, you will need to run the commands for the azure storage authorization process
-and then log in to the Crtex XSOAR Azure App or Self-Deployed Azure App.
-
+The Azure Storage content pack enables you to manage storage accounts and blob services and blob containers.
+
+## What does this pack do?
+
+The integration in this pack enables you to:
+- Get storage account details 
```

Comment:
is the change here intentional?

Reply:
No, that's a little weird because I think I did not modify this file.
If you notice the content is the same as the original file.